app/src/main/java/edu/grinnell/grinnell_publications_android/Fragments/ExpandedArticleFragment.java

```diff
@@ -0,0 +1,142 @@
+package edu.grinnell.grinnell_publications_android.Fragments;
+
+
+import android.graphics.PorterDuff;
+import android.graphics.drawable.Drawable;
+import android.support.annotation.Nullable;
+import android.os.Bundle;
+
+import android.support.design.widget.CollapsingToolbarLayout;
+import android.support.design.widget.FloatingActionButton;
+import android.support.design.widget.Snackbar;
+import android.support.v4.app.Fragment;
+import android.support.v4.content.ContextCompat;
+import android.support.v7.widget.Toolbar;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import edu.grinnell.grinnell_publications_android.Models.Interfaces.UserInterface;
+import edu.grinnell.grinnell_publications_android.R;
+
+import static android.support.design.widget.Snackbar.LENGTH_SHORT;
+
+
+/**
+ * Fragment displays articles and allows users to mark article as favorited.
+ * @author Yazan Kittaneh
+ */
+
+public class ExpandedArticleFragment extends Fragment implements UserInterface {
+
+    private ImageView mHeaderImage;
+    private CollapsingToolbarLayout mCollapsingToolbar;
+    private FloatingActionButton mFavoriteButton;
+    private TextView mArticleContent;
+    private Toolbar mArticleToolbar;
+
+    private boolean isFavorited = false;
```

Comment:
Nit-picking: You should just follow the same member variable naming convention here. This should read `private boolean mIsFavorited`